examples/basics/scene/viewbox.py

```diff
@@ -24,7 +24,7 @@
 # First ViewBox uses a 2D pan/zoom camera
 vb1 = scene.widgets.ViewBox(name='vb1', border_color='yellow',
                             parent=canvas.scene)
-vb1.clip_method = 'fbo'
+vb1.clip_method = 'fragment'
```

Comment:
What *should* be the default? (also in relation to the stereo.py example). Would 'viewport' not be the most efficient?